src/SFA.DAS.LevyTransferMatching.Web/Models/Pledges/ApplicationsViewModel.cs

```diff
@@ -12,5 +12,6 @@ public class ApplicationsViewModel : ApplicationsPostRequest
         public bool DisplayRejectedBanner { get; set; }
         public string RejectedEmployerName { get; set; }
         public bool RenderCreatePledgeButton { get; set; }
+        public bool RenderContinueButton { get; set; }
```

Comment:
I would call this "RenderRejectButton" or similar, for clarity

Reply:
This is now changed to "RenderRejectButton"